diem-unified-charter.md

```diff
@@ -1,12 +1,25 @@
 # Introduction
 
-Digital Emblems provide a mechanism for conveying a set of attributes
-bound to an asset that is presented to a validator and
-asserted by an issuer.
+An emblem is a device, symbol, or figure adopted and used as an identifying mark.
+As such, emblems provide a mechanism for conveying a set of attributes
+bound to an asset. An emblem may be presented to a validator and attributes are asserted by an issuer.
+
+In culture, emblems such as a flag, badge or coat of arms communicate group identity.
+In speech, emblems are specific nonverbal gestures or signals that have a direct verbal translation and are widely understood within a particular culture or community.
+The emblems of the International Committee of the Red Cross (ICRC)  are symbols of protection under the Geneva Conventions and are to be worn by all medical and humanitarian personnel and also displayed on their vehicles and buildings while they are in an active warzone, and all military forces operating in an active warzone must not attack entities displaying these emblems.
```

Comment:
```suggestion
The emblems of the International Committee of the Red Cross (ICRC)  are symbols of protection under the Geneva Conventions.
```

Reply:
This change would align with a comment made on the other PR.

Reply:
yes, it good to have it a bit shorter!